src/legacy/core_plugins/kibana/index.js

```diff
@@ -107,12 +107,12 @@ export default function(kibana) {
           category: DEFAULT_APP_CATEGORIES.analyze,
         },
         {
-          id: 'kibana:dev_tools',
+          id: 'kibana:devTools',
           title: i18n.translate('kbn.devToolsTitle', {
             defaultMessage: 'Dev Tools',
           }),
           order: 9001,
-          url: '/app/kibana#/dev_tools',
+          url: '/app/kibana#/devTools',
```

Comment:
Do not change the url, this is a breaking change

Reply:
@flash1293 if we don't change the url, we need to update the kibana route mapping somehow, because the kibana app url mapping is based on the plugin id:
https://github.com/elastic/kibana/blob/d73e15f4147d1de9a7ee15517cbf6a90f4d4b213/src/legacy/core_plugins/kibana/public/local_application_service/local_application_service.ts#L27

For now I don't see any appropriate property in the [App](https://github.com/elastic/kibana/blob/master/src/core/public/application/types.ts#L202) interface. As an option we can just add a condition for `devTools` plugin:
```
`/${
    typeof prefixOrApp === 'string'
      ? prefixOrApp
      : prefixOrApp.id === 'devTools'
      ? 'dev_tools'
      : prefixOrApp.id
  }/:tail*?`;
```
But maybe it's not the best solution.

Reply:
That's not the plugin id, it's the app id (as defined here: https://github.com/elastic/kibana/blob/master/src/plugins/dev_tools/public/plugin.ts#L99 ). The app id can stay `dev_tools`.

Reply:
Oh, I see. Thank you for the explanations. With the `dev_tools` app id it works :).